addons/docs/src/frameworks/angular/compodoc.ts

```diff
@@ -156,8 +157,17 @@ export const extractType = (property: Property, defaultValue: any) => {
 
 const extractDefaultValue = (property: Property) => {
   try {
-    // eslint-disable-next-line no-eval
-    const value = eval(property.defaultValue);
+    let value = property.defaultValue?.replace(/^'(.*)'$/, '$1');
+    if (value == null && property.jsdoctags.length > 0) {
+      property.jsdoctags.forEach((tag: JsDocTag) => {
+        if (['default', 'defaultvalue'].includes(tag.tagName.escapedText)) {
+          const tmp = window.document.createElement('DIV');
```

Comment:
Is this just to unescape HTML? If so, is there a library we can use for this?

Reply:
I originally used DOMParser code below but husky pre-commit throws this error - `'DOMParser' is not defined`. 
``` 
const dom = new DOMParser().parseFromString(tag.comment, 'text/html');
value = dom.body.textContent;
```
If I use window.DOMParser then Typescript has errors. If I add `// @ts-ignore` then all is well. WDYT?
```
 // @ts-ignore
const dom = new window.DOMParser().parseFromString(tag.comment, 'text/html');
value = dom.body.textContent;
```

Reply:
i like this better if it works 😁